templates/page.hbs

```diff
@@ -13,8 +13,13 @@
   </head>
   <body>
     {{!-- Header goes here --}}
-
+    
     <main>
+      {{#each header}}
+        {{{module this}}}
+        
+      {{/each}}
```

Comment:
It doesn't work, the helper `{{#each ...}}` is only to iterate, please replace this block (from line 16 to 21) by `{{{ module header }}}`

Reply:
Thanks you. Done.

Reply:
I have changed the block but my page does not  work. I get the page to show my header placing the block {{#each header}} {{{module header}}} {{/each}}.